videoframes.py

```diff
@@ -168,6 +180,18 @@ def setup_actions(self):
 
         self.context_menu.addMenu(quality_submenu)
 
+    def open_stream_in_browser(self, event):
+        url = ("http://www." if "www" not in self.stream.url else "http://") + self.stream.url
```

Comment:
Let's cast the stream URL to lowercase before querying it. If the user would for whatever reason enter an URL in capital letters or mixed letters, eg. "WWW.GOOGLE.SE", this would not be equal to "www.google.se".

```python
>>> "WWW.GOOGLE.SE" == "www.google.se"
False
>>>
```